tests/vite-app/vite.config.mjs

```diff
@@ -5,6 +5,7 @@ import {
   scripts,
   templateTag,
   optimizeDeps,
+  build,
```

Comment:
let's call this `emberAddonPrebuild` or `compatPrebuild`. using `build` here is too generic